hooks/pre/13-clear_pulpcore_content.rb

```diff
@@ -0,0 +1,18 @@
+# See bottom of the script for the command that kicks off the script
+require 'English'
+
+def reset
+  stop_services
+  start_services(['postgresql']) if local_postgresql?
+  empty_db_in_postgresql('pulpcore') if pulpcore_enabled?
```

Comment:
This whole reset doesn't make sense if pulpcore is not enabled, right? Essentially this whole function becomes a noop then.

Reply:
Moved the condition to the block that actually calls reset, and added in the corresponding place for the pulp reset as well.